core/modules/widgets/navigator.js

```diff
@@ -561,7 +561,7 @@ NavigatorWidget.prototype.handlePerformImportEvent = function(event) {
 		importReport = [];
 	// Add the tiddlers to the store
 	importReport.push($tw.language.getString("Import/Imported/Hint") + "\n");
-	// Let the loggers do their work. 
+	// Let the loggers do their work
```

Comment:
This comment is far too vague as we have no predefined concept of "loggers" in the TiddlyWiki code.

Reply:
What do you suggest?

Reply:
Actually there is already a $tw.Logger classs in the core (see syncer.js for example)

Reply:
It seems like a more appropriate comment would be something like "Give hooks an opportunity to modify the tiddlers before we process them"